python/paddle/tensor/math.py

```diff
@@ -7769,6 +7769,55 @@ def bitwise_right_shift_(
         return _C_ops.bitwise_right_shift_(x, y, is_arithmetic)
 
 
+def __lshift__(
+    x: Tensor,
+    y: Tensor | int,
+    is_arithmetic: bool = True,
+) -> Tensor:
+    if isinstance(y, int):
+        y = paddle.to_tensor(y, dtype=x.dtype)
+    elif isinstance(y, float):
+        raise TypeError("unsupported operand type(s) for <<:'float'")
```

Comment:
注意下异常信息规范，这些异常信息的打印建议也放到单测中捕获并断言是否正确。
![image](https://github.com/user-attachments/assets/31e651d8-a2e5-4174-bd40-19cb08d424ce)

```suggestion
        raise TypeError("unsupported operand type(s) for <<: 'float' and 'Tensor'")
```